src/main/java/org/candlepin/bind/PoolOpProcessor.java

```diff
@@ -90,7 +90,9 @@ private void setPoolQuantity(Map<Pool, Long> poolQuantities) {
             return;
         }
 
-        poolCurator.lock(poolQuantities.keySet());
+        poolCurator.lock(poolQuantities.keySet().stream()
+            .filter(x -> !x.isLocked())
```

Comment:
This was a bug in the original PR; the locked filter was erroneously applied and causes pools to be skipped in the Sat case.

Reply:
good catch! updated